crankshaft-engine/src/task/input.rs

```diff
@@ -70,9 +72,9 @@ impl Input {
     }
 
     /// Fetches the file contents via an [`AsyncRead`]er.
-    pub async fn fetch(&self) -> Vec<u8> {
+    pub async fn fetch(&self) -> Cow<'_, [u8]> {
         match &self.contents {
-            Contents::Literal(content) => content.as_bytes().to_vec(),
```

Comment:
Literal contents should probably be storing `Vec<u8>` too and not `String`.

I think this originally was based off of TES's schema, which I don't think handles binary inputs well, but our API shouldn't be limited to what TES supports.